src/Sylius/Bundle/AdminBundle/Resources/config/grids/shipment.yml

```diff
@@ -57,15 +51,6 @@ sylius_grid:
                         class: "%sylius.model.channel.class%"
             actions:
                 item:
-                    ship:
-                        type: apply_transition
-                        label: sylius.ui.ship
-                        icon: plane
-                        options:
-                            link:
-                                route: sylius_admin_shipment_ship
-                                parameters:
-                                    id: resource.id
-                            class: teal
-                            transition: ship
-                            graph: sylius_shipment
+                    ship_with_tracking_code:
```

Comment:
Hmmmm.... I would stay with `ship` 😄 you still can do it *without* a tracking code 🚀